http://cvs.sourceforge.net/viewcvs.py/gaim/gaim/src/protocols/jabber/jabber.c?r1=1.260&r2=1.261&sortby=date&diff_format=u
https://sourceforge.net/tracker/index.php?func=detail&aid=966125&group_id=235&atid=100235

===================================================================
RCS file: /cvsroot/gaim/gaim/src/protocols/jabber/jabber.c,v
retrieving revision 1.260
retrieving revision 1.261
diff -u -r1.260 -r1.261
--- gaim/gaim/src/protocols/jabber/jabber.c	2004/05/31 07:23:49	1.260
+++ gaim/gaim/src/protocols/jabber/jabber.c	2004/06/04 00:02:25	1.261
@@ -740,6 +740,7 @@
 {
 	JabberStream *js = gc->proto_data;
 
+	jabber_presence_send(gc, "unavailable", _("Logged out"));
 	jabber_send_raw(js, "</stream:stream>", -1);
 
 	if(js->gsc) {
